.coveragerc

```diff
@@ -1,3 +1,9 @@
 # .coveragerc to control coverage.py
 [run]
 branch = True
+include =
```

Comment:
Instead of `include`, use `source` for things like test and `source_pkgs` for the first-party package. https://github.com/aio-libs/yarl/blob/master/.coveragerc is a good example.